src/Features/Core/Portable/ExternalAccess/UnitTesting/API/IUnitTestingSolutionCrawlerRegistrationServiceAccessor.cs

```diff
@@ -2,12 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Microsoft.CodeAnalysis.Host;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
 {
-    internal interface IUnitTestingSolutionCrawlerRegistrationServiceAccessor : IWorkspaceService
+    internal interface IUnitTestingSolutionCrawlerServiceAccessor : IWorkspaceService
     {
+        void Reanalyze(Workspace workspace, IEnumerable<ProjectId> projectIds = null, IEnumerable<DocumentId> documentIds = null, bool highPriority = false);
```

Comment:
💡 Consider removing the arguments not needed, and making the remaining ones required.

Reply:
+1 But I think we can do this as a follow up change in master.